src/Models/restaurants.json

```diff
@@ -0,0 +1,141 @@
+[  
+    {
+        "restaurantName": "MBM Trinbago Meals",
+        "address": "Pool Road, Birmingham B66 3DB",
+        "phoneNumber": {
+            "landline": null, 
+            "mobile": "07792557204"
+        },
+        "cusineCategory": "Trinidadian, dine-in and takeaway", 
+        "tripAdvisorRating": 5, 
+        "website": null, 
+        "instgram": null,
+        "facebook": "@mbmmeals", 
+        "email":"melinda.williams@live.co.uk", 
+        "imageLinks": {
+            "facebookImages": "https://www.facebook.com/mbmmeals/photos/a.111542720538174/141008090924970/?type=3&theater",
+            "instagramImages": null
+        } 
+    },
+
+    {
+        "restaurantName":"Earth's Kitchen",
+        "address": "180 High Street Deritend Digbeth, Birmingham, B12 0LD",
+        "phoneNumber": {
+            "landline": "0121 572 6163", 
+            "mobile": null
+        },
+        "cusineCategory": "Plant based Caribbean, dine-in and takeaway", 
+        "tripAdvisorRating": 5, 
+        "website": null, 
+        "instgram": "earthskitchen_",
+        "facebook": "@earthskitchen3", 
+        "email": null, 
+        "imageLinks": {
+            "facebookImages": "https://www.facebook.com/pg/earthskitchen3/photos/?tab=album&album_id=387848864958011&ref=page_internal", 
+            "instagramImages": "https://www.instagram.com/earthskitchen_/?hl=en"    
+        }
+
+    },
+
+    { 
+        "restaurantName": "Devon House Jamaican Restaurant & Bar",
+        "address": "16 Hall Street Jewellery Quarter, Birmingham B18 6BS",
+        "phoneNumber": {
+            "landline": "0121 448 7203", 
+            "mobile": null
+        },
+        "cusineCategory": "Caribbean dine-in", 
+        "tripAdvisorRating": 4.5, 
+        "website": "https://www.devonhouse.co.uk/", 
+        "instgram": "devonhousejq",
```

Comment:
maybe a part from the user name, we could also add the full URL path of instagram, and facebook. It will be helpful when creating a link to an icon

Reply:
we could leave it as username and in react create a function that would append the username in the database to the end of the URL. would save repetition.

Reply:
Good thinking @AffirmedVisionary! The first part of the url will always be `https://instagram.com/` so that can hardcoded in, and then we just append the username at the end like you said, e.g. `https://instagram.com/${instagram}`. Wouldn't even need to store it in the database because it _should_ just be needed on the frontend to create the link